### PR TITLE
dbeaver/pro#3620 Introduce JUL handler

### PR DESCRIPTION
**Pull request by ivgag**
image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    @Override
+    public void publish(LogRecord record) {
+        log.log(
+                new Status(
+                        getSeverity(record.getLevel()),
+                        record.getLoggerName(),
+                        record.getMessage(),
+                        record.getThrown()
+                )
+        );
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java]:
```suggestion
        log.log(new Status(
            getSeverity(record.getLevel()),
            record.getLoggerName(),
            record.getMessage(),
            record.getThrown()
        ));
```

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Override
+    public void flush() {
+
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java]:
```suggestion
        // no op
```

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Override
+    public void close() throws SecurityException {
+
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java]:
```suggestion
        // no op
```

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+package org.jkiss.dbeaver;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+public class JULHandler extends Handler {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/JULHandler.java:29]: Please write a short Javadoc describing what this handler is used for.

**Comment by ShadelessFox** [comment]
I don't like that you're using INFO logging level in the driver. They bloat the Error Log view which is used for tracking issues like warnings and errors. Can it be fixed within a separate ticket?

**Review by ShadelessFox — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by ivgag** [comment]
> I don't like that you're using INFO logging level in the driver. They bloat the Error Log view which is used for tracking issues like warnings and errors. Can it be fixed within a separate ticket?

So, if you think that INFO logs bloat the Error Log view, which is meant for tracking warnings and errors, perhaps INFO logs shouldn’t be included in the Error Log? I don’t see a reason to avoid using the INFO logging level in the driver.

**Comment by E1izabeth** [comment]
Error Log view has Filters actually 
image

**Comment by Matvey16** [comment]
Verified

**Comment by serge-rider** [comment]
Anyway, I agree that spamming info messages in Error Log view is not a good idea.
Logging above looks more like debug log.

**Comment by Matvey16** [comment]
Verified

**Review by E1izabeth — APPROVED** [review]

**Pull request merged**